src/store/index.ts

```diff
@@ -0,0 +1,101 @@
+import { createDirectStore, ModulesImpl } from 'direct-vuex';
+import getUrlParameter from '@/helpers/get-url-parameter';
+import { IMessage } from '@/types/c-notification';
+import session from './modules/session';
+import notification from './modules/notification';
+import breadcrumb from './modules/breadcrumb';
+
+interface IModules {
+  session: ModulesImpl;
+  breadcrumb: ModulesImpl;
+  notification: ModulesImpl;
+}
+
+export interface IStore {
+  modules: IModules;
+}
+
+interface IAction {
+  data: object;
+  messages: {
+    INFO: [];
+    ERROR: [];
+    SUCCESS: [];
+  };
+  assetsPath: string;
+}
+
+interface IInitialData {
+  [key: string]: IAction;
```

Comment:
🤔 actually, we don't know what is inside IAction. In general it should be `{ data: {}, messages: [] }` but it actually does vary quite often. Your `IAction` interface is too SAP hybris specific.

We should not limit this for now. Maybe we can create a follow up ticket to discuss if we want to unify this.

Reply:
changed to any